protos/sources/vega/snapshot/v1/snapshot.proto

```diff
@@ -634,6 +634,7 @@ message PendingERC20MultisigControlSignature {
   string nonce = 3;
   uint64 epoch_seq = 4;
   bool added = 5;
+  string submitter_address = 6;
```

Comment:
Related to my other comment, the pending signatures aren't related to a submitter and you can have multiple submitters per-pending signatures. So I think we need this the `issue_signatures` part instead.